locales/en/yourloops.json

```diff
@@ -104,6 +104,7 @@
   "team-edit-dialog-placeholder-name": "Name",
   "team-edit-dialog-placeholder-phone": "Phone",
   "Cancel": "Cancel",
+  "common-close": "Close",
```

Comment:
```suggestion
  "close": "Close",
```
?

Reply:
As you wish

Reply:
On your other PR for de-duplicate the translation keys, I finally change my mind.
I will have probably set to "button-close" I think.
Well, maybe it's better to wait we have all the base code done to clean-up theses.